QRCoderTests/Helpers/HelperFunctions.cs

```diff
@@ -44,34 +45,53 @@ public static Bitmap BitmapSourceToBitmap(DrawingImage xamlImg)
                 }
             }
         }
-#endif 
+#endif
 
-#if !NETCOREAPP1_1
         public static string GetAssemblyPath()
         {
             return
-#if NET5_0 || NET6_0
-            AppDomain.CurrentDomain.BaseDirectory;
-#elif NET35 || NET452
-            Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().CodeBase).Replace("file:\\", "");
+#if NET5_0_OR_GREATER
+                AppDomain.CurrentDomain.BaseDirectory;
+#elif NETFRAMEWORK
+                Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().CodeBase).Replace("file:\\", "");
+#elif NETCOREAPP1_1
+                Path.GetDirectoryName(typeof(HelperFunctions).GetTypeInfo().Assembly.Location).Replace("file:\\", "");
 #else
-            Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location).Replace("file:\\", "");
+                Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location).Replace("file:\\", "");
 #endif
         }
-#endif
 
 
 #if !NETCOREAPP1_1
-        public static string BitmapToHash(Bitmap bmp)
+        /// <summary>
+        /// Converts a bitmap to a hash string based on the pixel data
+        /// using a deterministic algorithm that ignores compression algorithm
+        /// differences across platforms.
```

Comment:
❤️❤️❤️ Wow! Thanks - that problem was on my bucket list for years. (My development machine gave different hashes as the Github Action runner.) Glad to see this hopefully solved.